Resources/public/views/recommendations.html.twig

```diff
@@ -0,0 +1,31 @@
+<div id="recommendation-{{ recommendationId }}" class="ez-recommendation">
+    {{ 'Loading recommended items...'|trans }}
+</div>
+
+<script id="recommendation-template-{{ recommendationId }}" type="text/x-handlebars-template">
+    {{ template|raw }}
+</script>
+
+<script>
+    (function() {
+        var recoClient{{ recommendationId }} = new EzRecoRestClient({
+            scenarioId: '{{ scenarioId }}',
+            locationId: '{{ locationId }}',
+            limit: '{{ limit }}',
+            restUrl: '/recommendations/fetch' // TODO: dynamically replace with path from routing.yml
```

Comment:
Well, no, I'm never completely sure" :-)

But rather than looking for a reason *not to do it*, I look for a reason to do it. I don't see a reason why the developer would want to change this URI. The URI is determined by the service (YooChoose), and if the service changes, we will have to change it anyway. And it won't, because of BC.

And since in any case, the implementation can be extended, the developer does have the option to change it by extending the class and adding a new service of his own.